v0.3/tutorials/quickstart 1 button stopwatch.eve

```diff
@@ -0,0 +1,198 @@
+---
+layout: default
+title: Quickstart + One Button Stopwatch
+---
+
+{% raw %}
+
+# Quickstart + One Button Stopwatch
+
+This guide expands on the quickstart `Clock` page and shows you how to turn the clock into a one button stopwatch. The following code is meant to replace all the code from the original clock.
+
+## Implement system time and a race timer
+
+```
+commit
+  [#time #system/timer resolution: 10]
+  [#race-timer #stopped #reset]
+```
+
+Stopwatches tend to measure seconds to two decimal points, so our resolution changes to 10ms (or one centisecond) to account for that. The other thing added here is a `#race-timer` record that lets us keep track of when the stopwatch is stopped or running or has been reset back to zero.
```

Comment:
"stopped or running or has been reset back to zero." > "stopped, running, or reset."